File: Lesson_18/python_vs_javascript.js
// JavaScript version: same ideas as the Python file, different syntax.

// Print output
console.log("Hi from JavaScript");

// Variable
let age = 25;

// Constant
const MAX_AGE = 120;

// Function
function greet(name) {
    return "Hello, " + name + "!";
}

// If statement
if (age > 18) {
    console.log("Adult");
}

// Array
const skills = ["HTML", "CSS"];

// Object
const student = {
    name: "Sara",
    age: age,
    skills: skills
};

// Use the function and print the data
console.log(greet(student.name));
console.log(student);
console.log("Maximum age:", MAX_AGE);

File: Lesson_18/python_vs_javascript.py
# Python version: same ideas as the JavaScript file, different syntax.

# Print output
print("Hi from Python")

# Variable
age = 25

# Constant-style variable
# Python does not have a true const keyword. By convention, uppercase means
# "do not change this value".
MAX_AGE = 120

# Function
def greet(name):
    return "Hello, " + name + "!"


# If statement
if age > 18:
    print("Adult")

# List
skills = ["HTML", "CSS"]

# Dictionary
student = {
    "name": "Sara",
    "age": age,
    "skills": skills
}

# Use the function and print the data
print(greet(student["name"]))
print(student)
print("Maximum age:", MAX_AGE)
